app/src/main/java/org/mozilla/fenix/home/recenttabs/RecentTabsListFeature.kt

```diff
@@ -22,20 +22,17 @@ import org.mozilla.fenix.home.HomeFragmentStore
  */
 @OptIn(ExperimentalCoroutinesApi::class)
 class RecentTabsListFeature(
-    private val browserStore: BrowserStore,
+    browserStore: BrowserStore,
     private val homeStore: HomeFragmentStore
 ) : AbstractBinding<BrowserState>(browserStore) {
 
     override suspend fun onState(flow: Flow<BrowserState>) {
-        flow.map { it.selectedTabId }
```

Comment:
I think this was the only usage of `selectedTabId`. Should we also remove the extension function for it and its test since it has no consumers anymore?

Reply:
Spoke offline; this is a non-issue. `selectedTabId` is a property of `TabSessionState`.